### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by antonijap**
Hi. I had a lot of trouble with implementing this library since docs are outdated. It's not so hard to figure what to do once you dig into the code, but why not save some time for new users?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,80 +1,139 @@
 
+
 <p align="left">
-<!--<a href="https://travis-ci.org/EurekaCommunity/GooglePlacesRow"><img src="https://travis-ci.org/EurekaCommunity/GooglePlacesRow.svg?branch=master" alt="Build status" /></a>-->
 <img src="https://img.shields.io/badge/platform-iOS-blue.svg?style=flat" alt="Platform iOS" />
-<a href="https://developer.apple.com/swift"><img src="https://img.shields.io/badge/swift3-compatible-4BC51D.svg?style=flat" alt="Swift 3 compatible" /></a>
-<a href="https://cocoapods.org/pods/GooglePlacesRow"><img src="https://img.shields.io/cocoapods/v/GooglePlacesRow.svg" alt="CocoaPods compatible" /></a>
```
Comment (mats-claassen) [inline README.md]: Why do you remove this?

Comment (antonijap) [reply README.md]: It wasn't working on my preview. How do you want to proceed?

Comment (mats-claassen) [reply README.md]: It works for me if I copy it into Atom for example (with Markdown preview). I would want to leave as it shows the latest pod version on Cocoapods

**Comment by antonijap** [comment]
Now it should be okay like it was.

**Pull request merged**